client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -181,7 +181,7 @@ class TransferOtherUser extends React.Component {
 				},
 				{
 					action: 'confirm',
-					label: this.props.translate( 'Confirm Transfer' ),
+					label: this.props.translate( 'Confirm transfer' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.